packages/shared/src/consts.ts

```diff
@@ -8,38 +8,6 @@ const POOLS_NETWORK_FEE_HUNDREDTH_PIPS: Partial<
 export const getPoolsNetworkFeeHundredthPips = (network: ChainflipNetwork) =>
```

Comment:
think we can replace this with the implementation in `packages/swap/src/utils/rpc.ts`?